.github/workflows/promote.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
       - uses: actions/checkout@v4.1.7
       - name: Remove git edge stable
         run: |
+          git pull
```

Comment:
is this the same as https://github.com/actions/checkout?tab=readme-ov-file#fetch-all-history-for-all-tags-and-branches?

Reply:
lovely, updated!